Libraries/LibWeb/HTML/TraversableNavigable.h

```diff
@@ -154,6 +159,8 @@ class TraversableNavigable final : public Navigable {
     String m_window_handle;
 
     RefPtr<Gfx::SkiaBackendContext> m_skia_backend_context;
+    OwnPtr<Painting::DisplayListPlayerSkia> m_skia_player;
+    HashMap<Gfx::Bitmap*, NonnullRefPtr<Gfx::PaintingSurface>> m_bitmap_to_surface;
```

Comment:
I think it works for now, but it would be cool to figure how to replace `Web::Painting::BackingStore` with `Gfx::PaintingSurface` so there is no need for maintaining a mapping between them. I tried doing such refactoring a while ago and the problem was that we don't have a backend context by the time backing store is currently allocated IIRC.

Reply:
For all intents and purposes WebContent should probably directly spawn a context and then create the required PaintingSurfaces from there, yeah. Future PR!